tests/objcthemis/objthemis/SecureMessageTests.m

```diff
@@ -154,16 +165,27 @@ - (void)signVerifyWithPrivateKey:(NSData *)privateKeyData publicKey:(NSData *)pu
     NSString *resultString = [[NSString alloc] initWithData:decryptedMessage encoding:NSUTF8StringEncoding];
     XCTAssertTrue([message isEqualToString:resultString], @"initial string and decrypted string should be the same");
   
+    // NULL message
+    NSError *nullErrorEn = nil;
+    NSData *nullMessageEn = [encrypter wrapData:NULL error:&nullErrorEn];
+    XCTAssertNotNil(nullErrorEn, @"signing data with NULL message is error");
+    XCTAssertNil(nullMessageEn, @"signing data with NULL message should return nil message");
+    
+    NSError *nullErrorDe = nil;
+    NSData *nullMessageDe = [encrypter unwrapData:NULL error:&nullErrorDe];
+    XCTAssertNotNil(nullErrorDe, @"verifying data with NULL message is error");
+    XCTAssertNil(nullMessageDe, @"verifying data with NULL message should return nil message");
+    
     // empty message
-    NSError *themisEmptyErrorEn = nil;
-    NSData *emEmptyMessageEn = [encrypter wrapData:NULL error:&themisEmptyErrorEn];
-    XCTAssertNotNil(themisEmptyErrorEn, @"signing data with empty message is error");
-    XCTAssertNil(emEmptyMessageEn, @"signing data with empty message should return nil message");
-  
-    NSError *themisEmptyErrorDe = nil;
-    NSData *emEmptyMessageDe = [encrypter unwrapData:NULL error:&themisEmptyErrorDe];
-    XCTAssertNotNil(themisEmptyErrorDe, @"verifying data with empty message is error");
-    XCTAssertNil(emEmptyMessageDe, @"verifying data with empty message should return nil message");
+    NSError *emptyErrorSig = nil;
+    NSData *emptyMessageSig = [encrypter wrapData:NULL error:&emptyErrorSig];
```

Comment:
Ditto for empty data here and in the next block. Copy-paste is the root of all evil.

Reply:
NULL != empty, so now we test both